pymclevel/leveldbpocket.py

```diff
@@ -1843,7 +1854,11 @@ def add_data(self, terrain=None, tile_entities=None, entities=None, subchunk=Non
             ids_get = self.world.defsIds.mcedit_ids.get
             if DEBUG_PE:
                 write_dump(('\\' * 80) + '\nParsing Entities in chunk %s,%s\n' % (self.chunkPosition[0], self.chunkPosition[1]))
-            Entities = loadNBTCompoundList(entities)
+            try:
+                Entities = loadNBTCompoundList(entities)
+            except:
+                logger.error('potential entity corruption at chunk '+str(self.chunkPosition))
```

Comment:
Please, use the variable interpolation format like on line 1845/1856, or the `format` method of the string object.

Reply:
I am aware of this method however I have always done the lazy way. Feel free to reformat if you want